isis/src/control/objs/ControlNet/ControlNet.cpp

```diff
@@ -1112,7 +885,16 @@ namespace Isis {
    */
   QList< ControlMeasure * > ControlNet::GetValidMeasuresInCube(QString serialNumber) {
     ValidateSerialNumber(serialNumber);
-    return (*cubeGraphNodes)[serialNumber]->getValidMeasures();
+
+    QList< ControlMeasure * > validMeasures;
+
+    QList< ControlMeasure * > measureList = m_controlGraph[m_vertexMap[serialNumber]].measures;  
```

Comment:
We should just do a call to GetMeasuresInCube to generate the initial list

Reply:
:+1: Done.